### PR TITLE
Isogram: add camelCase test. Fixes #1303

### LINKED ISSUE #1303: isogram: no test for duplicate if the second one is upper case
I suggest adding this test (C track):
```
void test_duplicated_letter_mixed_case2(void)
{
   TEST_ASSERT_FALSE(is_isogram("alphAbet"));
}
```

### PR DESCRIPTION
**Pull request by wiredancer**

### CODE REVIEW

File: exercises/isogram/canonical-data.json [hunk 1]
```diff
@@ -59,6 +59,14 @@
           },
           "expected": false
         },
+        {
+          "description": "word with duplicated character in camel case",
+          "property": "isIsogram",
+          "input": {
+            "phrase": "alphAbet"
```
Comment (cmccandless) [inline exercises/isogram/canonical-data.json]: Incidentally, `isIsogram` is also a valid phrase for this test.

**Comment by Insti** [comment]
Mention fix in body so it's linkable: #1303
You will need to [update the version number.](https://github.com/exercism/problem-specifications#test-data-versioning)

File: exercises/isogram/canonical-data.json (hunk 1 above)
Comment (Insti) [reply exercises/isogram/canonical-data.json]: From the description it looks like `isIsogram` would be valid test.

Are the conditions:

* single letter duplicated
* first occurrence lowercase
* second occurrence uppercase

?

Perhaps the description could be modified to better explain the problem.

**Comment by rpottsoh** [comment]
Cleaned up bogus commits.

**Comment by wiredancer** [comment]
replaced by new PR #1307

**Pull request closed without merging**